platform/darwin/src/MGLOfflineStorage.h

```diff
@@ -174,6 +184,21 @@ MGL_EXPORT
  */
 @property (class, nonatomic, readonly) MGLOfflineStorage *sharedOfflineStorage;
 
+#pragma mark - Adding Contents of File
+
+/**
+ Adds the contents of the file path.
+ 
+ Once the content is added, then the `completion` block is executed with the
+ packs passed in.
+ 
```

Comment:
Also say something about the file format if possible, and that this method is used to preload pregenerated offline packs that are bundled with the application or downloaded manually by the application.